docs/content/en/docs/releases/v1.15.0-rc1.md

```diff
@@ -0,0 +1,81 @@
+Please refer documentation [here](/docs/usage/observability/opentelemetry/) for more details.
+
+Changes for [issue #1861](https://github.com/fission/fission/issues/1861) :
+
+- Initial integration [PR #2157](https://github.com/fission/fission/pull/2157)
+- Context passing enhancements [PR #2172](https://github.com/fission/fission/pull/2172) [PR #2175](https://github.com/fission/fission/pull/2175)
+- Capturing events within spans for important actions [PR #2180](https://github.com/fission/fission/pull/2180)
+- Support for OpenTelemetry standard environment variables [PR #2194](https://github.com/fission/fission/pull/2194)
+- Support for different propagators and samplers [PR #2201](https://github.com/fission/fission/pull/2201)
+
+### Redis Sets Keda Connector
+
+We have added a new Keda connector for Redis Sets. Please refer [here](/docs/usage/triggers/message-queue-trigger-kind-keda/redis/) for more details.
+
+## Regression fixes
+
+- Fixed ingress creation from Fission cli [PR #2164](https://github.com/fission/fission/pull/2164) [Issue #2144](https://github.com/fission/fission/issues/2144)
+- With prefix path support, function having common prefix were colliding. We have fixed that. [PR #2182](https://github.com/fission/fission/pull/2182) [Issue #2181](https://github.com/fission/fission/issues/2181) [Issue #2107](https://github.com/fission/fission/issues/2107)
+
+## Fixes
+
+- Changed default sync period for Fission informer to 30min from 30s. This reduces lot of unnecessary updates across Fission code [PR #2167](https://github.com/fission/fission/pull/2167)
+- Added filter for env update in newdeploy to consider newdeploy functions [PR #2165](https://github.com/fission/fission/pull/2165)
+- Changed default function call timeout from 30s to 60s in `fission fn test` [PR #2169](https://github.com/fission/fission/pull/2169)
+- Using typed informers in configmap and secret watcher [PR #2174](https://github.com/fission/fission/pull/2174)
+- Fixed pod name in port forwarding logs [PR #2178](https://github.com/fission/fission/pull/2178)
+- Added kubernetes objects aware logger [PR #2179](https://github.com/fission/fission/pull/2179)
+- Warn if package resource version mismatch in function specialization [PR #2183](https://github.com/fission/fission/pull/2183)
+
+### Internal Fixes
+
+- Updated Fission base image from Alpine 3.13 to Alpine 3.14. [PR #2160](https://github.com/fission/fission/pull/2160)
+- Updated Fission to use Go 1.16 and Kubernetes dependencies to 1.21 [PR #2168](https://github.com/fission/fission/pull/2168)
+- Stow client updated v0.2.7 [PR #2195](https://github.com/fission/fission/pull/2195)
+- Using goreleaser for faster buids and releases [PR #2189](https://github.com/fission/fission/pull/2189)
+
+## Note
+
+- From 1.15 release, Fission would be have having one tag only of format v1.15.x etc. instead of two tags earlier. Please ensure you are using v1.15.0 in helm install and not 1.15.0.
```

Comment:
Sorry what does this mean?

Reply:
Added example.

Reply:
Helm smart works with both version strings though.

```
-> % helm show values --version 1.15.0-rc1   fission-charts/fission-core | grep imageT
ag
imageTag: v1.15.0-rc1
  imageTag: v1.15.0-rc1
-> % helm show values --version v1.15.0-rc1   fission-charts/fission-core | grep imageTag
imageTag: v1.15.0-rc1
  imageTag: v1.15.0-rc1
```